<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HtmlEditor</class>
 <widget class="QMainWindow" name="HtmlEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_preview"/>
    <addaction name="actionExportPDF"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>F&amp;ormat</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionTextBold"/>
    <addaction name="actionTextItalic"/>
    <addaction name="actionTextUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionAlignLeft"/>
    <addaction name="actionAlignCenter"/>
    <addaction name="actionAlignRight"/>
    <addaction name="actionAlignJustify"/>
    <addaction name="separator"/>
    <addaction name="actionTextColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
  </widget>
  <widget class="QToolBar" name="fileToolbar">
   <property name="windowTitle">
    <string>File Actions</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="actionExportPDF"/>
  </widget>
  <widget class="QToolBar" name="editToolBar">
   <property name="windowTitle">
    <string>Edit Actions</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
  </widget>
  <widget class="QToolBar" name="formatToolbar">
   <property name="windowTitle">
    <string>Format Actions</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTextBold"/>
   <addaction name="actionTextItalic"/>
   <addaction name="actionTextUnderline"/>
   <addaction name="actionTextColor"/>
   <addaction name="actionAddImage"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionAlignJustify"/>
  </widget>
  <widget class="QToolBar" name="styleToolBar">
   <property name="windowTitle">
    <string>Format Actions</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/filenew.png</normaloff>:/CustomWidgets/Images/HtmlEditor/filenew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/fileopen.png</normaloff>:/CustomWidgets/Images/HtmlEditor/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/filesave.png</normaloff>:/CustomWidgets/Images/HtmlEditor/filesave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/fileprint.png</normaloff>:/CustomWidgets/Images/HtmlEditor/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Print...</string>
   </property>
   <property name="toolTip">
    <string>Print...</string>
   </property>
  </action>
  <action name="actionPrint_preview">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/fileprint.png</normaloff>:/CustomWidgets/Images/HtmlEditor/fileprint.png</iconset>
   </property>
   <property name="text">
    <string>Print preview...</string>
   </property>
   <property name="toolTip">
    <string>Print preview...</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/editundo.png</normaloff>:/CustomWidgets/Images/HtmlEditor/editundo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/editredo.png</normaloff>:/CustomWidgets/Images/HtmlEditor/editredo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/editcut.png</normaloff>:/CustomWidgets/Images/HtmlEditor/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/editcopy.png</normaloff>:/CustomWidgets/Images/HtmlEditor/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/editpaste.png</normaloff>:/CustomWidgets/Images/HtmlEditor/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="actionTextBold">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/textbold.png</normaloff>:/CustomWidgets/Images/HtmlEditor/textbold.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bold</string>
   </property>
  </action>
  <action name="actionTextItalic">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/textitalic.png</normaloff>:/CustomWidgets/Images/HtmlEditor/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Italic</string>
   </property>
  </action>
  <action name="actionTextUnderline">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/textunder.png</normaloff>:/CustomWidgets/Images/HtmlEditor/textunder.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Underline</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/textleft.png</normaloff>:/CustomWidgets/Images/HtmlEditor/textleft.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Left</string>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/textcenter.png</normaloff>:/CustomWidgets/Images/HtmlEditor/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;enter</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/textright.png</normaloff>:/CustomWidgets/Images/HtmlEditor/textright.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Right</string>
   </property>
  </action>
  <action name="actionAlignJustify">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/textjustify.png</normaloff>:/CustomWidgets/Images/HtmlEditor/textjustify.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Justify</string>
   </property>
  </action>
  <action name="actionTextColor">
   <property name="text">
    <string>&amp;Color...</string>
   </property>
   <property name="toolTip">
    <string>Color</string>
   </property>
  </action>
  <action name="actionExportPDF">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/exportpdf.png</normaloff>:/CustomWidgets/Images/HtmlEditor/exportpdf.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ExportPDF...</string>
   </property>
  </action>
  <action name="actionAddImage">
   <property name="icon">
    <iconset resource="../CustomWidgets_images.qrc">
     <normaloff>:/CustomWidgets/Images/HtmlEditor/add-image.png</normaloff>:/CustomWidgets/Images/HtmlEditor/add-image.png</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="toolTip">
    <string>Image</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../CustomWidgets_images.qrc"/>
 </resources>
 <connections/>
</ui>
